src/RankingMain.elm

```diff
@@ -0,0 +1,122 @@
+module RankingMain exposing (Model, Msg, init, update, view)
+
+import Browser
+import Colors exposing (black, blue, blueGreen, lightBlue, orange, purple, sky, white)
+import Element exposing (centerX, centerY, column, fill, height, html, layout, maximum, padding, rgb255, shrink, spacing, text, width)
+import Element.Background as Background
+import Element.Border as Border
+import Element.Font as Font exposing (center)
+import Element.Input as Input exposing (button)
+import Maybe
+import Ranking
```

Comment:
Where's `Ranking`?
You haven't committed it.

Reply:
will